sky/cli.py

```diff
@@ -1535,11 +1540,14 @@ def autostop(
     down: bool,  # pylint: disable=redefined-outer-name
     yes: bool,
 ):
-    """Schedule or cancel auto-stopping for cluster(s).
+    """Schedule or cancel an auto-stop or auto-down for cluster(s).
 
     CLUSTERS are the name (or glob pattern) of the clusters to stop.  If both
     CLUSTERS and ``--all`` are supplied, the latter takes precedence.
 
+    If --down is passed, autodown (tear down the cluster; non-restartable) is
+    used, rather than autostop (restartable).
```

Comment:
nit: be consistent everywhere in auto-verb vs autoverb. Either seems fine.

Reply:
Good point! Fixed. Thanks!